README.md

```diff
@@ -1,5 +1,7 @@
 # Draft Proposal for SES (Secure EcmaScript)
 
+**Superceded (at least in part) by the [Compartments](https://github.com/tc39/proposal-compartments) proposal.**
+
```

Comment:
Too strong. The `Compartment` parts are superceded. The rest is not. Especially `lockdown` and `harden`.